internal/controller/pkg/revision/runtime_function.go

```diff
@@ -192,3 +199,19 @@ func functionServiceOverrides() []ServiceOverride {
 		ServiceWithClusterIP(corev1.ClusterIPNone),
 	}
 }
+
+// getFunctionImage determines a complete function image, taking into account a
+// default registry. If the function meta specifies an image, we have a
+// preference for that image over what is specified in the package revision.
+func getFunctionImage(functionMeta *pkgmetav1beta1.Function, pr v1.PackageRevisionWithRuntime, defaultRegistry string) (string, error) {
```

Comment:
```suggestion
func getFunctionImage(fm *pkgmetav1beta1.Function, pr v1.PackageRevisionWithRuntime, defaultRegistry string) (string, error) {
```

This might be a good opportunity to switch to a shorter variable name. `functionMeta` seems redundant given it's mostly repeating information that already exists in the type. AFAICT it has limited scope in the sense discussed at https://github.com/crossplane/crossplane/tree/master/contributing#use-descriptive-variable-names-sparingly.

Reply:
Good call! I'll make the variable name shorter here and in a couple other similar places.